test/fast/Aliases/lts/"nvm alias" should ensure LTS alias dir exists

```diff
@@ -7,10 +7,23 @@ set -ex
 
 LTS_ALIAS_PATH="$(nvm_alias_path)/lts"
 
-die () { echo "$@" ; exit 1; }
+if [ -d "${LTS_ALIAS_PATH}" ]; then
```

Comment:
Do I need to  call `setup` and `teardown` after that? Or it'll be called each round?